todo_app/flask_config.py

```diff
@@ -4,5 +4,8 @@
 class Config:
```

Comment:
I don't think we're using this class any more, so we can remove it entirely now

Reply:
Updated,thank you